integrations/pom.xml

```diff
@@ -183,7 +183,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plain-credentials</artifactId>
-      <version>1.4</version>
+      <version>1.5-20180703.101522-1</version>
```

Comment:
I guess this was for https://github.com/jenkinsci/plain-credentials-plugin/pull/11? Then you should be using Incrementals, and you should wait for that PR to be merged before merging this one.